docs/index.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Main
+title: Patrol
```

Comment:
These titles are also used for analytics. If we change them, we'll break it (at least for some time)

Reply:
I think it's better to fix it earlier then.

Reply:
You can propose another title for this page as well, I'm not happy with the new one actually.

Reply:
> I think it's better to fix it earlier then.

Yeah, I agree. Let's just make sure that business is okay with it.

> You can propose another title for this page as well, I'm not happy with the new one actually.

TBH, I have no idea as well 😅 I don't like both names.

Reply:
`Patrol - Flutter testing framework`? It's the title of the home page - https://patrol.leancode.co

Reply:
`Patrol – Flutter UI test framework`

A tiny bit shorter and more descriptive.

Reply:
We decided to leave `Main` for now